lib/categories.js

```diff
@@ -1,8 +1,6 @@
-'use strict';
-
-const request = require('./utils/request');
-const cheerio = require('cheerio');
-const { BASE_URL } = require('./constants');
+import request from './utils/request.js';
+import { load } from 'cheerio';
```

Comment:
you should be able to do it like this, as per [the documentation](https://github.com/cheeriojs/cheerio#loading):

```
import * as cheerio from 'cheerio';

// ...

.then(cheerio.load)
```